RecoHI/HiTracking/python/HIInitialJetCoreClusterSplitting_cff.py

```diff
@@ -31,14 +31,15 @@
 from RecoLocalTracker.SiPixelRecHits.SiPixelRecHits_cfi import siPixelRecHits
 from RecoTracker.MeasurementDet.MeasurementTrackerEventProducer_cfi import MeasurementTrackerEvent
 from RecoPixelVertexing.PixelLowPtUtilities.siPixelClusterShapeCache_cfi import *
-hiInitialJetCoreClusterSplitting = cms.Sequence(
-                                hiPixelVerticesPreSplitting
-                                * hiCaloTowerForTrkPreSplitting
-                                * hiAkPu4CaloJetsForTrkPreSplitting
-								* hiAkPu4CaloJetsCorrectedPreSplitting
-				* hiAkPu4CaloJetsSelectedPreSplitting
-                                * hiJetsForCoreTrackingPreSplitting
-				* siPixelClusters
-                                * siPixelRecHits
-                                * MeasurementTrackerEvent
-                                * siPixelClusterShapeCache)
+hiInitialJetCoreClusterSplittingTask = cms.Task(
+                                hiPixelVerticesPreSplittingTask
+                                , hiCaloTowerForTrkPreSplitting
+                                , hiAkPu4CaloJetsForTrkPreSplitting
+				, hiAkPu4CaloJetsCorrectedPreSplitting
+				, hiAkPu4CaloJetsSelectedPreSplitting
+                                , hiJetsForCoreTrackingPreSplitting
+				, siPixelClusters
+                                , siPixelRecHits
+                                , MeasurementTrackerEvent
+                                , siPixelClusterShapeCache)
+hiInitialJetCoreClusterSplitting = cms.Sequence(hiInitialJetCoreClusterSplittingTask)
```

Comment:
This was only needed in RecoHI/HiTracking/python/HiTracking_cff.py: sequence not needed any more now, please remove